camera_matrix:           [ [6.0020210878700971e+02, 0.,                     2.9555439707618132e+02],
                         [  0.,                     6.0016122703033739e+02, 2.2302358884115333e+02],
                         [  0.,                     0.,                     1.                    ] ]

distortion_coefficients: [ -2.7643750662487081e-02, -2.0043746890891415e-01, -2.8336818488409665e-03, -8.8473122176714134e-03, 6.1086926330839253e-01 ]
